release-notes/v1_72.md

```diff
@@ -101,6 +101,58 @@ The following improvements to shell integration were made:
 - Removed the bash script's usage of the `$PREFIX` environment variable which could conflict with some build tools.
 - Polished the zsh script to fix some edge cases around `$ZDOTDIR`.
 - Better handling of zsh shell integration on machines that are shared between multiple users.
+- Quick fix, see below
+
+### Quick fix
+
+We're starting to roll out a new feature powered by shell integration called quick fixes.
```

Comment:
Shall we move this to the top since it's the coolest new feature, then move `Shell integration improvements` to second since it's related?

Reply:
Yes was thinking that too but didn't want to usurp your spot 😄 or have complex merge